docker/openemr/flex-edge/autoconfig.sh

```diff
@@ -51,6 +60,8 @@ auto_setup() {
     if [ "$EASY_DEV_MODE" != "yes" ]; then
         chmod -R 600 /var/www/localhost/htdocs/openemr
     fi
+    
+    chmod -R 600 .
```

Comment:
looks like this slipped in (guessing from copy/paste from the 5.0.3 docker code)